v1/summary.yaml

```diff
@@ -109,39 +109,43 @@ paths:
               body: '{{storage.body}}'
 
         # Storage miss. Call the Action API to get the textextract.
-        - extract:
+        - get_extracts:
+            request:
+              method: get
+              uri: '{{options.host}}/{domain}/v1/page/summary/{title}'
+          get_properties:
             request:
               method: post
               uri: /{domain}/sys/action/query
               body:
-                prop: 'info|extracts|pageimages|revisions|pageterms|coordinates'
-                exsentences: 5
-                exintro: true
+                prop: 'info|pageimages|revisions|pageterms|coordinates'
                 piprop: 'thumbnail|original'
                 inprop: 'displaytitle'
                 pithumbsize: 320
                 pilicense: 'any'
                 rvprop: 'timestamp|ids'
                 titles: '{{request.params.title}}'
                 wbptterms: 'description'
-            response:
+        - extract:
+            return:
               # Define the response to save & return.
+              status: 200
               headers:
-                content-type: application/json; charset=utf-8; profile="https://www.mediawiki.org/wiki/Specs/Summary/1.2.0"
-                etag: "\"{{getRevision(extract.body.items[0].revisions).revid}}/{{timeuuid()}}\""
+                content-type: "{{get_extracts.headers.content-type}}"
+                etag: "{{get_extracts.headers.etag}}"
               body:
-                title: '{{extract.body.items[0].title}}'
-                displaytitle: '{{extract.body.items[0].displaytitle}}'
-                pageid: '{{extract.body.items[0].pageid}}'
-                extract: '{{stripTags(extract.body.items[0].extract)}}'
-                extract_html: '{{extract.body.items[0].extract}}'
-                thumbnail: '{{httpsSource(extract.body.items[0].thumbnail)}}'
-                originalimage: '{{httpsSource(extract.body.items[0].original)}}'
-                lang: '{{extract.body.items[0].pagelanguagehtmlcode}}'
-                dir: '{{extract.body.items[0].pagelanguagedir}}'
-                timestamp: '{{getRevision(extract.body.items[0].revisions).timestamp}}'
-                description: '{{extractDescription(extract.body.items[0].terms)}}'
-                coordinates: '{{processCoords(extract.body.items[0].coordinates)}}'
+                title: '{{get_properties.body.items[0].title}}'
+                displaytitle: '{{get_properties.body.items[0].displaytitle}}'
+                pageid: '{{get_properties.body.items[0].pageid}}'
+                extract: '{{get_extracts.body.extract}}'
+                extract_html: '{{get_extracts.body.extract_html}}'
```

Comment:
The `get_extract` response also has a `type` property. I think we'd want to expose this one as well. At least for the new version.

Reply:
@jdlrobson also added a new `revision` property. Not sure about that one since it could also be retrieved from the etag but probably doesn't hurt.

Reply:
Done Done